tests/system/providers/amazon/aws/redshift/example_redshift_data_execute_sql.py

```diff
@@ -22,7 +22,11 @@
 from airflow.decorators import task
 from airflow.providers.amazon.aws.hooks.redshift_data import RedshiftDataHook
 from airflow.providers.amazon.aws.operators.redshift_data import RedshiftDataOperator
+from airflow.utils.trigger_rule import TriggerRule
+from tests.system.providers.amazon.aws.utils import set_env_id
 
+ENV_ID = set_env_id()
+DAG_ID = 'example_redshift_cluster'
 REDSHIFT_CLUSTER_IDENTIFIER = getenv("REDSHIFT_CLUSTER_IDENTIFIER", "redshift_cluster_identifier")
 REDSHIFT_DATABASE = getenv("REDSHIFT_DATABASE", "redshift_database")
 REDSHIFT_DATABASE_USER = getenv("REDSHIFT_DATABASE_USER", "awsuser")
```

Comment:
Please create these resources during a test setup phase (using env id to keep resource identifiers unique). If any resources are unreasonable to create in the test then please use [fetch_variable](https://github.com/ferruzzi/airflow/blob/18896156aac8c62c3fe098b546ee601ff386da98/tests/system/providers/amazon/aws/utils/__init__.py#L105) instead of getenv (this allows test configuration to live in SSM Parameterstore for easier test automation).